libraries/physics/src/ObjectMotionState.h

```diff
@@ -140,6 +140,11 @@ class ObjectMotionState : public btMotionState {
 
     bool isActive() const { return _body ? _body->isActive() : false; }
 
+    bool hasInternalKinematicChanges() const { return _hasInternalKinematicChanges; }
+
+    void dirtyInternalKinematicChanges() { _hasInternalKinematicChanges = true; }
+    void clearInternalKinematicChanges() { _hasInternalKinematicChanges = false; }
+
```

Comment:
I gave these methods long obnoxious names (rather than **setDirtyFlags()** or something) so that no one would misuse them for other purposes.  These names reflect exactly what they are used for and nothing more.